src/Main.cpp

```diff
@@ -12,24 +14,30 @@ struct Movement
 	geometry::Vec2 force;
 	F32            moveforce;
 
-	Movement(F32 moveforce, F32 velocitymax) : velocity(0.0f, 0.0f), moveforce(moveforce), velocitymax(velocitymax), force(0.0f, 0.0f) { }
+	Movement(F32 moveforce, F32 velocitymax) 
+		: velocity(0.0f, 0.0f)
+		, moveforce(moveforce)
```

Comment:
I think I left this comment somewhere else in this team as well but let's use `_` in between words in variable names. Change this to `move_force` etc.